src/server/graphql/mutations/helpers/createNewMeetingPhases.ts

```diff
@@ -29,6 +29,32 @@ const primePhases = (phases: GenericMeetingPhase[]) => {
   secondPhase.stages[0].isNavigableByFacilitator = true
 }
 
+const getPastStageDurations = async (teamId: string) => {
+  const r = getRethink()
+  return r
+    .table('NewMeeting')
+    .getAll(teamId, {index: 'teamId'})
+    .filter({isLegacy: false}, {default: true})
+    // .orderBy(r.desc('endedAt'))
+    .concatMap((row) => row('phases'))
+    .concatMap((row) => row('stages'))
+    .filter((row) => row.hasFields('startAt', 'endAt'))
+    // convert seconds to ms
+    .merge((row) => ({
```

Comment:
This is a great query

Reply:
It makes me wonder if we shouldn't be emitting segment events per stage anyway so we could easily interrogate this stuff in amplitude (vs. needing to poke the database)